### ISSUE TITLE
KF image lags when scrolling inside GeometryReader

### ISSUE THREAD

**Issue by ForceGT** [onevcat/Kingfisher#1621]
The code snippet is this

```
ScrollView(.vertical){
            
            VStack(alignment: .leading){
                //HEADER IMAGE
                GeometryReader{ geometry in
                    
                    if geometry.frame(in: .global).minY > -360{
                        
                        KFImage(URL(string: wallpaper.image ?? Config.DUMMY_URL))
                            .blur(radius: CGFloat(blurRadius), opaque: true)
                            .aspectRatio(contentMode: showImage ? .fit :.fill)
                            .offset(y: (-geometry.frame(in: .global).minY))
                            .frame(width: UIScreen.main.bounds.width,
                                   height: geometry.frame(in: .global).minY > 0 ? geometry.frame(in: .global).minY + 360 : 360, alignment: .center)
                            .onTapGesture {
                                withAnimation(.easeInOut(duration: 0.75)) {
                                    showImage.toggle()
                                }
                            }
                    }
                }// :HEADER IMAGE
                .frame(width:UIScreen.main.bounds.width,height: 360,alignment: .center)
```

**Comment by ForceGT** [comment]
Posting a screen record just in case 

https://user-images.githubusercontent.com/31688051/107320658-28c57f00-6ac7-11eb-80c1-6ba97c313fdf.mov

**Comment by onevcat** [comment]
@ForceGT Not sure how this can happen. 

Recently we had a few changes on how to identify the loaded image, which is possible to this issue. Maybe it worths trying the `master` branch to see if there is any difference?

Thanks!

**Comment by justintime06** [comment]
Using the master branch worked for me, thank you so much!!!!

**Comment by onevcat** [comment]
The same fixes should be already released as a part of 6.1.1. I am closing it now. If the issue is still there, feel free to reopen it.
